src/client/components/Overview/Overview.styles.css

```diff
@@ -0,0 +1,42 @@
+.overview {
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 2em;
+}
+.chatTitle {
+  width: 90%;
+  height: 20%;
+  margin: 10px;
+  text-align: center;
+}
+.users {
```

Comment:
Its not your job to write the css for the components lucy and tariq did, they do that themselves